src/server/transforms.js

```diff
@@ -0,0 +1,246 @@
+var async = require('async');
+var os = require('os');
+var packageJson = require('../../package.json');
+var parser = require('./parser');
+var requestIp = require('request-ip');
+var url = require('url');
+var _ = require('../utility');
+
+var defaultSettings = {
+  host: os.hostname(),
+  environment: 'development',
+  framework: 'node-js',
+  showReportedMessageTraces: false,
+  notifier: {
+    name: 'node_rollbar',
```

Comment:
This should probably be "rollbar.js" but it might require some changes in the pipeline.